index.js

```diff
@@ -8,6 +8,16 @@ function ToObject(val) {
 	return Object(val);
 }
 
+function ownEnumerableKeys(obj) {
+	var keys = Object.getOwnPropertyNames(obj);
+	if (Object.getOwnPropertySymbols) {
+		keys = keys.concat(Object.getOwnPropertySymbols(obj));
+	}
+	return keys.filter(function(key) {
```

Comment:
So, a space between `function` and parameters for anonymous functions but not for named functions, correct?